ocp_resources/priority_class.py

```diff
@@ -7,3 +7,34 @@ class PriorityClass(Resource):
     """
 
     api_group = Resource.ApiGroup.SCHEDULING_K8S_IO
+
+    def __init__(
+        self,
+        name,
+        client,
+        teardown=True,
+        value=None,
+        global_default=False,
+        description=None,
+        preemption_policy=None,
+    ):
+        super().__init__(
+            name=name,
+            client=client,
+            teardown=teardown,
+        )
+        self.value = value
+        self.global_default = global_default
+        self.description = description
+        self.preemption_policy = preemption_policy
+
+    def to_dict(self):
+        res = super().to_dict()
+        res.update({"value": self.value})
```

Comment:
Do you want to update value even when it's None? (this is the default)

Reply:
Value should be a mandatory field, I updated it to None because when 'get' called, it use the init of the requested resource

Reply:
I was sure this is a mandatory field, but seems like if not chosen the default is 0
I'll update (all fields must be None beside name because when 'get' called, it use the init of the requested resource)